Packages/TensorFlow-Examples.pck.st

```diff
@@ -804,25 +836,27 @@ initialize
 initializeGraph
 	graph _ TF_Graph create! !
 
-!MNIST3LayersNNExamplePlan methodsFor: 'initialization' stamp: 'gera 2/2/2017 02:03:56'!
+!MNIST3LayersNNExamplePlan methodsFor: 'initialization' stamp: 'JB 2/11/2017 14:21:08'!
 initializeInferenceGraph
 	| hidden1 hidden2 |
-	prediction _ graph fromBlock: [:image |
+	prediction _ graph fromBlock: [:image | | axis |
 		input _ image.
 		hidden1 _ (image * weights1 + biases1) rectified.
 		hidden2 _ (hidden1 * weights2 + biases2) rectified.
-		hidden2 * weights3 + biases3].
+		netInput _ hidden2 * weights3 + biases3.
```

Comment:
why is it called netInput? it's roughly at then of the forward path, I'd intuitively say it's closer to being the output of the net, than an input...